### PR TITLE
faq.js: Add the missing question and its suitable answer

### PR DESCRIPTION
**Pull request by yash10p**
Fixes https://github.com/coala/projects/issues/122

### CODE REVIEW

File: data/faq/steal_project_idea.md [hunk 1]
```diff
@@ -0,0 +1 @@
+The whole point of 'Open Source Software' is to encourage "stealing". If you do not honor original authors, licenses, and the "creator" of an original idea, there is no shame in saying "he had a great idea, I think I can implement it". On the plus side, we will always take into account who came up with something and if you help others succeed, we'll definitely credit you for that.
```
Comment (sims1253) [inline data/faq/steal_project_idea.md]: This is not correct.
If you do not honor them (by not saying they had the idea first) that is wrong.
If you follow license rules, disclaim original idea creators and authors you probably are fine. Usually talking to the involved parties is another good idea.

Could you limit the lines to 80 chars?

**Comment by sims1253** [comment]
also this is a feature not a bug fix so please use Closes instead of Fixes (see coala.io/commit for the commit message guidelines)

File: data/faq/steal_project_idea.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+The whole point of 'Open Source Software' is to encourage "stealing".
```
Comment (sils) [inline data/faq/steal_project_idea.md]: use consistent quoting please

File: data/faq/steal_project_idea.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+If you do honor original authors, licenses, and the "creator" of an
+original idea, there is no shame in saying "he had a great idea, I think
+I can implement it". It's called "collaboration". On the plus side, we will
+always take into account who came up with something and if you help
+others succeed, we'll definitely credit you for that.
```
Comment (sils) [inline data/faq/steal_project_idea.md]: I'd add another paragraph explaining that we essentially want to work with people who can work together, not with "lone hackers".

File: data/faq/steal_project_idea.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+The whole point of "Open Source Software" is to encourage "stealing".
```
Comment (hemangsk) [inline data/faq/steal_project_idea.md]: That 'stealing' part was just a symbolism, lets take it in right taste? sth like 
```
The whole point of "Open Source Software" is to encourage taking code from places, using and improving it.
```
```
The whole point of "Open Source Software" is to encourage freedom to use any code in the world, without obligations
```
```
The whole point of "Open Source Software" is to encourage celebration of complete independence in sharing and writing code
```

File: data/faq/steal_project_idea.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+The whole point of "Open Source Software" is to encourage celebration of
+complete independence in sharing and writing code.If you do honor original
+authors, licenses, and the "creator" of an original idea, there is no shame in
+saying "he had a great idea, I think I can implement it". It's called
+"collaboration". On the plus side, we will always take into account who came
+up with something and if you help others succeed, we'll definitely credit you
+for that. Most importantly, we want our contributors to work as a team and not
+like some "lone hackers" keeping secrets from fellow contributors, and sending
+unnecessary private messages to the project mentors.
```
Comment (Mixih) [inline data/faq/steal_project_idea.md]: Please point out that these is such a thing as stealing in OS, it is when you don't credit resources, authors, or disregard the license.

File: data/faq/steal_project_idea.md [hunk 6]
```diff
@@ -0,0 +1,9 @@
+The whole point of "Open Source Software" is to encourage celebration of
+complete independence in sharing and writing code.If you do honor original
```
Comment (Mariatta) [inline data/faq/steal_project_idea.md]: Add space before `If you do`

**Comment by sims1253** [comment]
could you polish this asap so we have it done before the application deadline?

**Comment by yash10p** [comment]
Sorry. I was out for a while. Maybe someone else can look into this issue now.

**Comment by sims1253** [comment]
cherry-picked into https://github.com/coala/projects/pull/209

**Pull request closed without merging**